src/pytorch_lightning/utilities/logger.py

```diff
@@ -132,7 +132,9 @@ def _sanitize_params(params: Dict[str, Any]) -> Dict[str, Any]:
     return params
 
 
-def _add_prefix(metrics: Dict[str, float], prefix: str, separator: str) -> Dict[str, float]:
+def _add_prefix(
```

Comment:
relevant to using `metrics: Dict[str, Union[Tensor, float]]` across all loggers for `metrics`.

Reply:
Hi @JustinGoheen, because of this change, this PR now doesn't pass `mypy` check

Reply:
updated; apologies for not fixing this earlier.